<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1b1d23;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="4,0,0">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>41</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="Import_button">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    appearance: button;
    background-color: #405cf5;
    border-radius: 6px;
    box-shadow: rgba(50, 50, 93, .1) 0 0 0 1px inset, rgba(50, 50, 93, .1) 0 2px 5px 0, rgba(0, 0, 0, .07) 0 1px 1px 0;
    color: #fff;
    cursor: pointer;
	font-size: 15px;
    font-family: -apple-system, system-ui, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica, Neue&quot;, Ubuntu, sans-serif;
    font-weight: bold;
    height: 30px;
    margin: 0 20px 0 20px;
    text-align: center;
    padding: 5px 10px; /* Adjust the values as needed for top/bottom and left/right padding */
}

QPushButton:hover {
    background-color: #283999;
}

QPushButton:pressed {
    background-color: #1c2973;
}
</string>
        </property>
        <property name="text">
         <string>Import DICOM File</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QRadioButton" name="surface_button">
          <property name="styleSheet">
           <string notr="true">color:white;
font-weight:bold;
font-size: 15px;
</string>
          </property>
          <property name="text">
           <string>Surface Rendering</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="raycast_button">
          <property name="styleSheet">
           <string notr="true">color:white;
font-weight:bold;
font-size: 15px;
</string>
          </property>
          <property name="text">
           <string>Raycast Rendering</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="IsoValue">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color:white;
font-weight:bold;
font-size: 15px;</string>
          </property>
          <property name="text">
           <string>Iso Value</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>7</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="clear_button">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    appearance: button;
    background-color: #405cf5;
    border-radius: 6px;
    box-shadow: rgba(50, 50, 93, .1) 0 0 0 1px inset, rgba(50, 50, 93, .1) 0 2px 5px 0, rgba(0, 0, 0, .07) 0 1px 1px 0;
	font-size: 15px;
    color: #fff;
    cursor: pointer;
    font-family: -apple-system, system-ui, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica, Neue&quot;, Ubuntu, sans-serif;
    font-weight: bold;
    height: 30px;
    margin: 0 20px 0 20px;
    text-align: center;
    padding: 5px 10px; /* Adjust the values as needed for top/bottom and left/right padding */
}

QPushButton:hover {
    background-color: #283999;
}

QPushButton:pressed {
    background-color: #1c2973;
}
</string>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QOpenGLWidget" name="openGLWidget">
      <property name="styleSheet">
       <string notr="true">border: 3px solid rgb(17,17,17);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(39, 44, 54);
border-radius: 10px;
border: none;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
